consensus/istanbul/core/core.go

```diff
@@ -422,3 +428,17 @@ func PrepareCommittedSeal(hash common.Hash) []byte {
 	buf.Write([]byte{byte(msgCommit)})
 	return buf.Bytes()
 }
+
+// Minimum required number of consensus messages to proceed
+func requiredMessageCount(valSet istanbul.ValidatorSet) int {
+	size := valSet.Size()
+	switch size {
+	// in the certain cases we must receive the messages from all consensus nodes to ensure finality...
+	case 1, 2, 3:
+		return int(size)
+	case 6:
+		return 2*valSet.F() + 2 // when the number of valSet is 6 and return value is 2*F+1, the return value is not safe. It should return 4 or more.
```

Comment:
How about returning 5 to be clear?

Reply:
Thanks for your review.
I reflected in https://github.com/klaytn/klaytn/pull/1462/commits/cada14266f5abdd9a61f120b8b5903c62401d0b0.